rdr_service/tools/tool_libs/genomic_utils.py

```diff
@@ -2000,6 +2003,86 @@ def run(self):
         return 0
 
 
+class GemToGpMigrationClass(GenomicManifestBase):
+    """
+    Load a GEM to GP Migration batch
+    """
+
+    def __init__(self, args, gcp_env: GCPEnvConfigObject):
+        super(GemToGpMigrationClass, self).__init__(args, gcp_env)
+
+        self.qr_dao = QuestionnaireResponseDao()
+        self.gem_gp_dao = GemToGpMigrationDao()
+
+    def run(self):
+
+        # Activate the SQL Proxy
+        self.gcp_env.activate_sql_proxy()
+
+        with GenomicJobController(GenomicJob.GEM_GP_MIGRATION_EXPORT,
+                                  bq_project_id=self.gcp_env.project) as controller:
+
+            results = self.get_data_for_export(controller)
+
+            self.export_to_gem_gp_table(controller, results)
+
+        return 0
+
+    def get_data_for_export(self, controller):
+        with self.qr_dao.session() as session:
+            results = session.query(
+                QuestionnaireResponse.participantId,
+                QuestionnaireResponse.authored,
+                Code.value
+            ).join(
+                QuestionnaireResponseAnswer,
+                QuestionnaireResponseAnswer.questionnaireResponseId == QuestionnaireResponse.questionnaireResponseId
+            ).join(
+                Code,
+                Code.codeId == QuestionnaireResponseAnswer.valueCodeId
+            ).outerjoin(
+                GemToGpMigration,
+                and_(GemToGpMigration.participant_id == QuestionnaireResponse.participantId,
+                     GemToGpMigration.run_id == controller.job_run)
+            ).filter(
+                Code.value.in_(["ConsentAncestryTraits_Yes",
+                                "ConsentAncestryTraits_No",
+                                "ConsentAncestryTraits_NotSure"])
+            )
+            if self.args.limit:
+                results = results.limit(self.args.limit)
+
+            return results.all()
+
+    def export_to_gem_gp_table(self, controller, results):
+        batch = []
+        batch_size = 1000
+
+        now_str = clock.CLOCK.now().strftime('%Y%m%d%H%M%S')
+        file_path = f"gem_gp_export_{now_str}.csv"
+
+        for row in results:
+            obj = self.gem_gp_dao.prepare_obj(row, controller.job_run, file_path)
+
+            batch.append(obj)
+
+            # write to table in batches
+            if len(batch) % batch_size == 0:
+                if not self.args.dryrun:
+                    print(f'Inserting batch starting with: {batch[0].participantId}')
```

Comment:
:nit Can we chaneg these to `_logger.info` statements.

Reply:
Yep, changed it.